CodeForReview.js

```diff
@@ -0,0 +1,33 @@
+var itemList = [];
+
+function addToList(itm) {
+  itemList.push(itm);
+  console.log("Item added: " + itm);
+  document.getElementById('list').innerHTML += '<li>' + itm + '</li>';
+}
+
+function checkAndRemoveFromList() {
+  var itemToRemove = document.getElementById('itemInput').value; 
+
+  if (itemList.indexOf(itemToRemove) > -1) { 
```

Comment:
Better to store indexOf in a separate var so you don't have to call the function each time
```suggestion
  if (itemList.indexOf(itemToRemove) > -1) { 
```